src/main/java/com/ifactory/press/db/solr/HitCount.java

```diff
@@ -40,14 +48,29 @@
     public ValueSource parse(FunctionQParser fp) throws SyntaxError {
         // hitcount() takes no arguments.  If we wanted to pass a query
         // we could call fp.parseNestedQuery()
+        IndexReader emptyReader = null;
+        try {
+            emptyReader = new MultiReader();
+        } catch (IOException ex) {
+            Logger.getLogger(HitCount.class.getName()).log(Level.SEVERE, null, ex);

```

Comment:
Perhaps we should be constructing a logger as a class attribute and reusing it, rather than continually calling `getLogger()`. Is there any downside to doing it that way? Seems like it would make this a lot more readable.